libs/ledger-live-common/src/bridge/jsHelpers.ts

```diff
@@ -81,14 +81,22 @@ export type GetAccountShapeArg0 = {
 };
 
 export type GetAccountShape = (
-  arg0: GetAccountShapeArg0,
-  arg1: SyncConfig
+  infos: AccountShapeInfos,
+  sysConfig: SyncConfig
 ) => Promise<Partial<Account>>;
-type AccountUpdater = (arg0: Account) => Account;
+type AccountUpdater = (account: Account) => Account;
 
+/**
+ * @WARNING
+ * @DEPRECATED due to circular dependency
```

Comment:
what is the circular dependency here ?

Reply:
Quoting myself from slack:

> I'm having an issue that hopefully someone can help me with...
I'm writing a unit test (weird, I know. :trollpika:) on a file that uses the infamous `bridge/jsHelpers` file, and that file also imports (as a far children dependency) the `bridge/impl.ts` which is responsible for dealing with one of our generated file merging in one file all bridges (`live-common/src/generated/bridge/js.ts` & `live-common/src/generated/bridge/mock.ts`).
> After that long list of deps, I can finally explain the issue :phew-obama:
> So running `pnpm jest src/families/evm/__tests__/logic.unit.test.ts` in live-common is resulting with the error (shared in the image).
> And the `makeScanAccount` that is undefined is coming from... `bridge/jsHelpers` :beaten-trollface: So basically full circle...
> logic.unit.test.ts -> jsHelpers -> ... -> generated/bridge/js.ts -> jsHelpers (now undefined) -> ...
<img width="578" alt="screenshot_2023-01-13_at_12 19 57" src="https://user-images.githubusercontent.com/44363395/213757675-e80b4ea6-4047-4f99-8925-eedd9fcaf2d3.png">